go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/google/go-cmp v0.5.6
 	github.com/madlambda/spells v0.4.0
-	github.com/mineiros-io/terramate v0.1.6
+	github.com/mineiros-io/terramate v0.1.20-0.20220802190017-bb5d27cd8ea9
```

Comment:
this points to https://github.com/mineiros-io/terramate/pull/512
needs to be changed to next release (when it's merged).